api/prometheus/api.go

```diff
@@ -280,8 +280,10 @@ func (qr *queryResult) UnmarshalJSON(b []byte) error {
 type QueryAPI interface {
 	// Query performs a query for the given time.
 	Query(ctx context.Context, query string, ts time.Time) (model.Value, error)
-	// Query performs a query for the given range.
+	// QueryRange performs a query for the given range.
 	QueryRange(ctx context.Context, query string, r Range) (model.Value, error)
+	// LabelValues performs a query for the values of the given label.
+	LabelValues(ctx context.Context, label string) (model.LabelValues, error)
```

Comment:
Valid concern, and one that we are violating by the page menu hierarchy under which https://prometheus.io/docs/querying/api/ is embedded as well.

I don't like the idea of `QueryAPI`, `LabelsAPI`, etc. much, but if we also want to add Alertmanager API functionality here eventually, then maybe `Prometheus` and `Alertmanager`? I'm not even sure whether we should keep the `...API` suffix. Or alternatively, `...Client` instead of `...API`.